pages/_app.js

```diff
@@ -40,6 +40,26 @@ export default function App({ Component, pageProps }) {
       mutate();
     }
   }
+  async function toggleHasCooked(_id) {
+    if (user.recipeInteractions.find((i) => i.recipe._id === _id)) {
```

Comment:
```suggestion
    if (user.recipeInteractions.find((interaction) => i.recipe._id === _id)) {
```
it nicer and more readable it you use the whole word and not just an letter :)